client/my-sites/domains/domain-management/dataviews/use-fields.tsx

```diff
@@ -0,0 +1,152 @@
+				),
+			},
+			{
+				id: 'owner',
+				label: translate( 'Owner' ),
+				enableHiding: false,
+				enableSorting: true,
+				getValue: ( { item }: { item: PartialDomainData } ) => {
+					const domain = getFullDomain( item );
+					if ( ! domain ) {
+						return '';
+					}
+					// Removes the username that appears in parentheses after the owner's name.
+					// Uses $ and the negative lookahead assertion (?!.*\() to ensure we only match the very last parenthetical.
+					return domain.owner.replace( / \((?!.*\().+\)$/, '' );
+				},
+				render: ( { item }: { item: PartialDomainData } ) => {
+					const domain = getFullDomain( item );
+					if ( ! domain ) {
+						return <LoadingPlaceholder />;
+					}
+					if ( ! domain.owner ) {
+						return '-';
+					}
+					return domain.owner.replace( / \((?!.*\().+\)$/, '' );
+				},
+			},
+			{
+				id: 'site',
+				label: translate( 'Site' ),
+				enableHiding: false,
+				enableSorting: true,
+				getValue: ( { item }: { item: PartialDomainData } ) => sites[ item.blog_id ]?.name ?? '',
+				render: ( { item }: { item: PartialDomainData } ) => {
+					const domain = getFullDomain( item );
+					if ( isLoadingSites || ! domain ) {
+						return <LoadingPlaceholder />;
+					}
+
+					const site = sites[ item.blog_id ];
+					const userCanAddSiteToDomain = domain?.currentUserCanCreateSiteFromDomainOnly ?? false;
+
+					return (
+						<DomainsTableSiteCell
+							site={ site }
+							userCanAddSiteToDomain={ userCanAddSiteToDomain }
+							siteSlug={ getSiteSlug( item ) }
+							domainName={ domain.domain }
+							hasConnectableSites={ hasConnectableSites }
+						/>
+					);
+				},
+			},
+			{
+				id: 'ssl_status',
+				label: translate( 'SSL' ),
+				enableHiding: false,
+				enableSorting: true,
+				getValue: ( { item }: { item: PartialDomainData } ) => {
+					const domain = getFullDomain( item );
+					return domain?.sslStatus || '';
+				},
+				render: ( { item }: { item: PartialDomainData } ) => <SslStatusField domain={ item } />,
+			},
+			{
+				id: 'expiry',
+				label: translate( 'Expires/Renews on' ),
+				enableHiding: false,
+				enableSorting: true,
+				getValue: ( { item }: { item: PartialDomainData } ) =>
+					item.expiry ? Date.parse( item.expiry ) : 0,
+				render: ( { item }: { item: PartialDomainData } ) => (
+					<DomainsTableExpiresRenewsOnCell domain={ item } as="div" />
+				),
+			},
+			{
+				id: 'domain_status',
+				label: translate( 'Status' ),
+				enableHiding: false,
+				enableSorting: true,
+				header: domainsRequiringAttention ? (
+					<>
+						{ translate( 'Status' ) }
+						<span className="list-status-cell__bubble">{ domainsRequiringAttention }</span>
+					</>
+				) : (
+					translate( 'Status' )
+				),
+				getValue: ( { item }: { item: PartialDomainData } ) => item.domain_status?.status,
+				render: ( { item }: { item: PartialDomainData } ) => <DomainStatusField domain={ item } />,
+			},
+		],
+		[
+			domainsRequiringAttention,
```

Comment:
`hasConnectableSites` is also a dependency we have to list here.

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/97677/commits/91bececa44af9c57eda117e22f2c27510f60e08d